site2/docs/reference-pulsar-admin.md

```diff
@@ -2333,17 +2342,252 @@ Remove the message TTL for a topic.
 Usage
 ```bash
 $ pulsar-admin topics remove-message-ttl tenant/namespace/topic
-=======
+
+Options  
```

Comment:
@sijia-w have you built locally to see the changes?

Reply:
I didn't add 'Options' inside the code block, but trying to fix the issues already existed.

Reply:
You can refine it as suggested, move "Options" outside the codeblock.